x-pack/plugins/security_solution/server/lib/timeline/utils/common.ts

```diff
@@ -45,6 +45,16 @@ export const throwErrors = (createError: ErrorFactory) => (errors: rt.Errors) =>
   throw createError(formatErrors(errors).join('\n'));
 };
 
+export class CustomBadRequestError extends Error {
```

Comment:
I would put this in a generic utils that aren't specific to timeline since we are consuming it from the `detection_engine`. There should be a generic utils higher up but in all _honesty_ we should avoid throwing too much into utils/helpers as that's starting to have a bad determinatal effect of pulling in a _lot_ of imports during testing and other issues. 

I would suggest we just take this and put it in its own single file at `security_solution/server/lib/` or even `security_solution/server` or we could even have a folder called `utils` such as `security_solution/server/utils` and put this there.

Any of those would be a lot better IMHO.

Reply:
done 7615a9f2cac7e8ef25f30b4a204d6c13c88313a7